tile-generation/src/main/scala/com/oculusinfo/tilegen/datasets/TilingTask.scala

```diff
@@ -477,6 +477,20 @@ class StaticTilingTask[PT: ClassTag, DT: ClassTag, AT: ClassTag, BT]
 					(index, value, analysis)
 				}
 			)
+
+			// If set, filter bins that are out of the level 0 tile bounds
+			if (config.filterToRegion) {
+				val iScheme = indexer.indexScheme
+				val area = getTilePyramid.getTileBounds(new TileIndex(0, 0, 0))
+
+				mappedData.filter(lineSeq => {
+					val (x, y) = iScheme.toCartesian(lineSeq._1)
+
+					y >= area.getMinY && y < area.getMaxY
```

Comment:
We need to check x here too, not just y.  WebMercator doesn't care much about x, everything else does.